x-pack/plugins/observability_solution/apm/server/routes/services/get_service_instance_metadata_details.ts

```diff
@@ -66,11 +67,12 @@ export async function getServiceInstanceMetadataDetails({
               filter: filter.concat({ term: { [METRICSET_NAME]: 'app' } }),
             },
           },
+          fields: ['*'],
```

Comment:
let's make the fields explicit as used in the `serviceInstanceMetadataDetailsMapping` function